alembic/versions/9569b2f8e588_add_e164_formatted_number_for_contacts.py

```diff
@@ -0,0 +1,135 @@
+"""Add E164 formatted number for contacts
+
+Revision ID: 9569b2f8e588
+Revises: 2adc8aff56ea
+
+"""
+
+import phonenumbers
+import sqlalchemy as sa
+from phonenumbers import NumberParseException, PhoneNumberFormat, is_valid_number
+from sqlalchemy import sql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '9569b2f8e588'
+down_revision = '2adc8aff56ea'
+
+
+contact_field_table = sql.table(
+    'dird_contact_fields',
+    sql.column('id'),
+    sql.column('name'),
+    sql.column('value'),
+    sql.column('contact_uuid'),
+)
+
+contact_table = sql.table(
+    'dird_contact',
+    sql.column('uuid'),
+    sql.column('phonebook_uuid'),
+    sql.column('user_uuid'),
+)
+
+phonebook_table = sql.table(
+    'dird_phonebook',
+    sql.column('uuid'),
+    sql.column('tenant_uuid'),
+)
+
+user_table = sql.table(
+    'dird_user',
+    sql.column('user_uuid'),
+    sql.column('tenant_uuid'),
+)
+
+tenant_table = sql.table(
+    'dird_tenant',
+    sql.column('uuid'),
+    sql.column('country'),
+)
+
+
+def get_tenants_country():
+    tenants = sa.select([tenant_table.c.uuid, tenant_table.c.country])
+    return {tenant.uuid: tenant.country for tenant in op.get_bind().execute(tenants)}
+
+
+def get_contacts_tenant():
+    phonebooks_tenant = (
+        contact_field_table.join(
+            contact_table,
+            contact_field_table.c.contact_uuid == contact_table.c.uuid,
+        )
+        .join(
+            phonebook_table,
+            contact_table.c.phonebook_uuid == phonebook_table.c.uuid,
+        )
+        .select()
+        .where(contact_field_table.c.name == 'number')
```

Comment:
What about columns name "fax", "mobile", "numéro", "num", "exten", etc
We are missing a concept on phonebooks and personal contacts that would allow dird to know which column is a number and which column is not.